nameserver/dns_test.go

```diff
@@ -1,39 +1,68 @@
 package nameserver
 
 import (
+	"fmt"
 	"net"
 	"testing"
+	"time"
 
 	"github.com/miekg/dns"
 	"github.com/stretchr/testify/require"
-	. "github.com/weaveworks/weave/common"
+
+	"github.com/weaveworks/weave/net/address"
+	"github.com/weaveworks/weave/router"
 )
 
-// Check that we can prune an answer
-func TestPrune(t *testing.T) {
-	EnableDebugLogging(testing.Verbose())
-	Log.Infoln("TestPrune starting")
-
-	questionMsg := new(dns.Msg)
-	questionMsg.SetQuestion("name", dns.TypeA)
-	questionMsg.RecursionDesired = true
-	question := &questionMsg.Question[0]
-	records := []ZoneRecord{
-		Record{"name", net.ParseIP("10.0.1.1"), 0, 0, 0},
-		Record{"name", net.ParseIP("10.0.1.2"), 0, 0, 0},
-		Record{"name", net.ParseIP("10.0.1.3"), 0, 0, 0},
-		Record{"name", net.ParseIP("10.0.1.4"), 0, 0, 0},
+func TestLargeUDPResonse(t *testing.T) {
```

Comment:
Renamed (in https://github.com/weaveworks/weave/commit/95e7ffc3d680a8852300a4a88038aa7d5dd3dcb3)